widget/embedded/src/constants/searchParams.ts

```diff
@@ -4,4 +4,5 @@ export enum SearchParams {
   'TO_CHAIN' = 'toBlockchain',
   'TO_TOKEN' = 'toToken',
   'FROM_AMOUNT' = 'fromAmount',
+  'REFERRER' = 'referrer',
```

Comment:
Please rename `referrer` to `auto connect` to be more explicit and what we internally call the feature.

Reply:
fixed